webapp/src/components/shareBoardComponent.tsx

```diff
@@ -56,7 +56,7 @@ const ShareBoardComponent = React.memo((props: Props): JSX.Element => {
         const accept = window.confirm(intl.formatMessage({id: 'ShareBoard.confirmRegenerateToken', defaultMessage: 'This will invalidate previously shared links. Continue?'}))
         if (accept) {
             const newSharing: ISharing = sharing || createSharingInfo()
-            newSharing.token = Utils.createGuid()
+            newSharing.token = Utils.createGuid('')
```

Comment:
I guess this id should be a token id

Reply:
Fixed via https://github.com/mattermost/focalboard/pull/1369/commits/03daaf722a0bff201708cf27b0434c8fbfef4f4c